packages/forms/frontend/sirius-components-widget-reference/src/modals/BrowseModal.tsx

```diff
@@ -30,51 +30,69 @@ const useBrowserModalStyles = makeStyles()((_) => ({
 
 export const BrowseModal = ({ editingContextId, widget, onClose }: BrowseModalProps) => {
   const { classes: styles } = useBrowserModalStyles();
-  const [browserSelection, setBrowserSelection] = useState<Selection>({ entries: widget.referenceValues });
+  const [browserSelection, setBrowserSelection] = useState<string[]>(widget.referenceValues.map((value) => value.id));
+
+  const onTreeItemClick = (event: React.MouseEvent<HTMLDivElement, MouseEvent>, item: GQLTreeItem) => {
+    if (widget.reference.manyValued) {
+      if (event.ctrlKey || event.metaKey) {
+        event.stopPropagation();
+        if (browserSelection.includes(item.id)) {
+          setBrowserSelection((prevState) => ({
+            ...prevState,
+            selectedTreeItemIds: prevState.filter((itemId) => itemId !== item.id),
+          }));
+        } else {
+          setBrowserSelection((prevState) => ({
+            ...prevState,
+            selectedTreeItemIds: [...prevState, item.id],
+          }));
```

Comment:
The data structure has changed (from selection to array of string) so this is a bug

Reply:
Done